adagrams/game.py

```diff
@@ -1,11 +1,112 @@
+import random
+from random import shuffle
+from collections import Counter
+
 def draw_letters():
-    pass
+    letters_pool = {
+        "A" : 9,
+        "B" : 2,
+        "C" : 2,
+        "D" : 4,
+        "E" : 12,
+        "F" : 2,
+        "G" : 3,
+        "H" : 2,
+        "I" : 9,
+        "J" : 1,
+        "K" : 1,
+        "L" : 4,
+        "M" : 2,
+        "N" : 6,
+        "O" : 8,
+        "P" : 2,
+        "Q" : 1,
+        "R" : 6,
+        "S" : 4,
+        "T" : 6,
+        "U" : 4,
+        "V" : 2,
+        "W" : 2,
+        "X" : 1,
+        "Y" : 2,
+        "Z" : 1
+    }
+    letters_probability = []
+    letter_bank = []
+    total = sum(letters_pool.values())
+    for letter in letters_pool:
+        p = letters_pool[letter] / total
+        letters_probability.append(p)
+    letters = list(letters_pool)
+
+    while len(letter_bank) < 10:
+        get_letter = random.choices(letters, weights=letters_probability, k=1)
+        letter_count = letter_bank.count(get_letter[0])
+        if letter_count < letters_pool[get_letter[0]]:   # Mistake: TypeError: unhashable type:"list"; because random() return a list.
```

Comment:
I'm curious about this error! But without context, I'm unsure if this comment should be deleted or expanded.